src/components/Footer/Footer.module.css

```diff
@@ -0,0 +1,11 @@
+.footer {
+   height: 20vh;
+   font-family: Roboto;
```

Comment:
You don't need Roboto as now it is under index.css if you remember.

I will say it again, I don't know if you missed it or what, but make sure to give yourself a CR before you ask someone else, so you can catch these kind of things :)